ecs-cli/modules/utils/compose/convert_task_definition.go

```diff
@@ -136,6 +136,9 @@ func convertToContainerDef(inputCfg *adapter.ContainerConfig, ecsContainerDef *C
 
 	// Set Linux Parameters
 	outputContDef.LinuxParameters = &ecs.LinuxParameters{Capabilities: &ecs.KernelCapabilities{}}
+	if inputCfg.Devices != nil {
```

Comment:
Nit: Can we move this to after setting the `Capabilities` field, just to match the ordering in the aws-sdk-go api?